twitter-langchain/README.md

```diff
@@ -1 +1,111 @@
 # Twitter (X) Langchain Toolkit
+CDP integration with Langchain to enable agentic workflows using the core primitives defined in `cdp-agentkit-core`.
+
+This toolkit contains tools that enable an LLM agent to interact with the [Coinbase Developer Platform](https://docs.cdp.coinbase.com/). The toolkit provides a wrapper around the CDP SDK and Twitter (X), allowing agents to perform social operations like posting text.
```

Comment:
This is not a CDP integration really. It's just a Twitter (X) integration with langchain as a toolkit. It is only in our monorepo for development convenience and if we can get the X team to own this eventually it would be a win.

Let's remove CDP langauge from the README.